source/anti-patterns/language.md

```diff
@@ -42,6 +42,10 @@ in {
 }
 ```
 
+:::{note}
+It is fine to use `rec` in nixpkgs' stdenv.mkDerivation function especially for reusing version in a fetcher for src since that is a fixed output derivation anyway.
```

Comment:
Should be discouraged a bit more, it's not great but the best we have in some cases.